crates/runtime/tests/crypto.rs

```diff
@@ -401,6 +401,45 @@ async fn crypto_decrypt_aes_cbc() {
     );
 }
 
+#[tokio::test]
+async fn crypto_decrypt_aes_ctr() {
```

Comment:
Could we split this test into `crypto_decrypt_aes_ctr` & `crypto_encrypt_aes_ctr`?